### DISCUSSION TITLE
How to make separate paths

### DISCUSSION THREAD

**Discussion by johnchourajr** [steveruizok/perfect-freehand#62] [Q&A]
Trying to use the example on Code Sandbox, which when pointerDown and pointerMove are used, each time a new path is created, how can I have those previous paths preserved while creating new?

**Comment by nate-lumalabs** [comment]
the example code uses a single list of points, and each time onPointerDown is called, it re-creates that list.

You have to modify the code to store a list of lists of points, better described as a list of strokes.
